File: 01-Classes/Animal.cpp
#include <iostream>
#include <string>

using namespace std;

class Animal {
public:
    string name;
    string species;

    Animal(string name, string species) {
        this->name = name;
        this->species = species;
    }

    void displayInfo() const {
        cout << "Name: " << name << ", Species: " << species << endl;
    }
};

int main() {
    Animal myAnimal1("Buddy", "Dog");
    myAnimal1.displayInfo();

    Animal myAnimal2("Madam", "Cat");
    myAnimal2.displayInfo();
    return 0;
}

File: 01-Classes/Animal.java
class Animal {
    String name;
    String species;

    public Animal(String name, String species) {
        this.name = name;
        this.species = species;
    }

    public void displayInfo() {
        System.out.println("Name: " + name + ", Species: " + species);
    }
    
    public static void main(String[] args) {
        Animal myAnimal1 = new Animal("Buddy", "Dog");
        myAnimal1.displayInfo();

        Animal myAnimal2 = new Animal("Madam", "Cat");
        myAnimal2.displayInfo();
    }
}
